### PR TITLE
Add top Elrond assets (ESDT / SFT) - part 1

### PR DESCRIPTION
**Pull request by andreibancioiu**
This PR adds some (10) of the top Elrond assets (ESDT / SFT).

The assets should be integrated with respect to the newly added functionality in TrustWallet Core on PR trustwallet/wallet-core#1845.

For each asset, `info.json` includes the type which - for the assets in this PR - can either be `esdt` (Elrond Standard Digital Token) or `sft` (Elrond Semi-Fungible Token).

## Type `ESDT`

In order to list the ESDT assets of an account, the following endpoint should be used:

```
https://api.elrond.com/accounts/{account}/tokens
```

(Optional) In order to fetch details of a ESDT asset, one can use the following endpoint:

```
https://api.elrond.com/tokens/{identifier}
```

In order to perform ESDT transfers, one should use the following .proto message from TW Core: `ESDTTransfer`.

## Type `ESDTSFT`

In order to list the SFT (and NFT, for that matter) assets of an account, the following endpoint should be used:

```
https://api.elrond.com/accounts/{account}/nfts
```

The response also contains the `nonce` field, which should be used when initiating SFT transfers.

(Optional) In order to fetch details of a ESDTSFT asset, one can use the following endpoint:

```
https://api.elrond.com/collections/{identifier}
```

In order to perform SFT transfers, one should use the following .proto message from TW Core: `ESDTNFTTransfer.`

## Notes

In order to check the requirement on number of holders & number of transactions, please perform a GET against:

```
https://api.elrond.com/tokens/{token identifier}/accounts/count
https://api.elrond.com/tokens/{token identifier}/transactions/count
```

### CODE REVIEW

**Comment by leedaniil** [comment]
Please, pull master into your branch and remove "api" field from info.json files.
You also need to add a few links to some of these resources: github, twitter, coingecko, coinmarketcap, discord, t.me or reddit.
Add at least 1 tag for each token.

**Comment by leedaniil** [comment]
Please, pull master into your branch and remove "api" field from info.json files.
You also need to add a few links to some of these resources: github, twitter, coingecko, coinmarketcap, discord, t.me or reddit.
Add at least 1 tag for each token.

File: blockchains/elrond/assets/EGLDUSDCF-8600f8/info.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "type": "ESDTSFT",
+    "id": "EGLDUSDCF-8600f8",
+    "name": "EGLDUSDCLPStaked",
```
Comment (vikmeup) [inline blockchains/elrond/assets/EGLDUSDCF-8600f8/info.json:4]:
```suggestion
    "name": "EGLD USDC/LP Staked",
```

Would it make more sense to make it this way?

File: blockchains/elrond/assets/USDC-c76f1f/info.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "type": "ESDT",
+    "id": "USDC-c76f1f",
+    "name": "WrappedUSDC",
```
Comment (vikmeup) [inline blockchains/elrond/assets/USDC-c76f1f/info.json:4]:
```suggestion
    "name": "Wrapped USDC",
```

File: blockchains/elrond/assets/USDC-c76f1f/info.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "symbol": "USDC",
+    "decimals": 6,
+    "status": "active",
+    "description": "USDC stablecoin originating on Ethereum, bridged as an ESDT token on Elrond. 1 USDC = 1 WrappedUSDC",
```
Comment (vikmeup) [inline blockchains/elrond/assets/USDC-c76f1f/info.json:8]:
```suggestion
    "description": "USDC stablecoin originating on Ethereum, bridged as an ESDT token on Elrond. 1 USDC = 1 Wrapped USDC",
```

Same for description

File: blockchains/elrond/assets/WEGLD-bd4d79/info.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+    "type": "ESDT",
+    "id": "WEGLD-bd4d79",
+    "name": "WrappedEGLD",
+    "symbol": "WEGLD",
```
Comment (vikmeup) [inline blockchains/elrond/assets/WEGLD-bd4d79/info.json:5]:
```suggestion
    "symbol": "wEGLD",
```

is it `wEGLD` or `WEGLD` ?

Comment (andreibancioiu) [reply blockchains/elrond/assets/WEGLD-bd4d79/info.json:5]: It's `WEGLD` - I've fixed the description.

File: blockchains/elrond/assets/EGLDUSDCF-8600f8/info.json (hunk 1 above)
Comment (andreibancioiu) [reply blockchains/elrond/assets/EGLDUSDCF-8600f8/info.json:4]: Renamed.

File: blockchains/elrond/assets/USDC-c76f1f/info.json (hunk 2 above)
Comment (andreibancioiu) [reply blockchains/elrond/assets/USDC-c76f1f/info.json]: Fixed.

File: blockchains/elrond/assets/USDC-c76f1f/info.json (hunk 3 above)
Comment (andreibancioiu) [reply blockchains/elrond/assets/USDC-c76f1f/info.json]: Fixed.

**Pull request merged**